lesson-17/index.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+
+// HW-17
+
+// task 1
+
+// const a = ("'true'");
+// const b = Boolean(NaN);
+// const c = (17);
+// const d = undefined;
+// const e = null;
+// console.log(typeof a);
+// console.log(typeof b);
+// console.log(typeof c);
+// console.log(typeof d);
+// console.log(typeof e);
+
+
+// task 2
+
+// const height = 15;
+// const width = 20;
+
+// if (height > width) {
+//     console.log(height);
+// } else if (width > height) {
+//     console.log(width);
+// }
+
+
+// task 3
+
+// for (let i = 1; i <= 20; i+=1) {
+//     if (i % 3 == 0) {
+//       console.log(i);
+//     }
+// }
+
+
+// task 4
+
+// const key = true;
+// const documents = true;
+// const pen = true;
+// const apple = false;
+// const orange = true;
+// let shouldGoToWork = (key && documents && pen && apple || orange);
```

Comment:
1. Скобки вокруг не нужны.
2. У каждого оператора есть приоритет операции. && более приоритетная операция, поэтому запуститься раньше чем ||. У тебя логически некорректно условие, хоть ответ правильный.
![изображение](https://user-images.githubusercontent.com/32203047/157731538-313c4bd3-c93f-4b1e-8f71-9a9e39bc1591.png)

Правильнее будет 
```suggestion
// let shouldGoToWork = key && documents && pen && (apple || orange);
```